webpack/src/universitas/components/Story/StoryFoot.js

```diff
@@ -5,23 +5,32 @@ import Advert from 'components/Advert'
 
 const STROSSLE_ID = 'dff15dfe-e8ca-4e6d-b547-8038ab88562b'
 
+const ACCELERATOR_ID = 'cf6209fc-8eae-4041-bbf9-6d9129336326'
+
 class StrossleWidget extends React.Component {
   constructor(props) {
     super(props)
     const strossleMount = () => {
       global.strossle && global.strossle(STROSSLE_ID, '.strossle-widget')
     }
-    this.componentDidMount = strossleMount
-    this.componentDidUpdate = strossleMount
+
+   const acceleratorMount = () => {
+     global.strossle && global.strossle(ACCELERATOR_ID, '.strossle-widget')
+   }
+
+    this.componentDidMount = strossleMount && acceleratorMount
+    this.componentDidUpdate = strossleMount && acceleratorMount
   }
   shouldComponentUpdate(nextProps) {
     return nextProps.url != this.props.url
   }
 
   render() {
-    return <div className="strossle-widget" />
+    return ( 
+          <div className="strossle-widget"/>
```

Comment:
There should be two widgets here with different class names. For example: 

```
<> 
  <div className="strossle-widget-related" />
  <div className="strossle-widget-accelerator" data-spklw-widget="cf6209fc-8eae-4041-bbf9-6d9129336326" />
</>
```
The Strossle v2 widget uses the `className` (class) to figure out which div to use, but the v1 widget uses `data-spklw-widget` instead.

Reply:
And per the explanation from strossle, we should also insert the story url here. That might work. I don't really know.  
```
<div data-spklw-widget="cf6209fc-8eae-4041-bbf9-6d9129336326" data-spklw-url={this.props.url} />
```

And pass in `url` as a prop to the StrosseleWidget component.